src/tracks/playabletrack/notetrack/ui/NoteTrackAffordanceControls.cpp

```diff
@@ -127,7 +127,8 @@ void NoteTrackAffordanceControls::Draw(TrackPanelDrawingContext& context, const
             context.dc.SetTextBackground(wxTransparentColor);
             context.dc.SetTextForeground(theTheme.Colour(clrClipNameText));
             context.dc.SetFont(wxFont(wxFontInfo()));
-            TrackArt::DrawClipAffordance(context.dc, clipRect, nt->GetName(), highlight, selected);
+            const auto titleRect = TrackArt::DrawClipAffordance(context.dc, clipRect, highlight, selected);
+            TrackArt::DrawClipTitle(context.dc, titleRect, nt->GetName());
```

Comment:
Should analogous changes be made for note tracks?  It might make sense but there would be less demand for such corrections.

Reply:
Not sure if I understand your question correctly, but there is no midi clip name editing feature yet.